src/arrayMethodSort.js

```diff
@@ -5,7 +5,36 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let count;
+
+    const compareAsString = (a, b) => {
+      if (String(a) > String(b)) {
+        return 1;
+      } else if (String(a) === String(b)) {
+        return 0;
+      } else {
+        return -1;
+      }
+    };
```

Comment:
You can simplify:
```js
function customCompare(a, b) {
  return a.toString() > b.toString();
}

```
And make use of default function parameter for compareFunction param

Reply:
Thank you for this comment :). => fixed!